user-module/controller/user-controller.js

```diff
@@ -88,6 +80,7 @@ class UserController {
       logger.info('Admin setting user email verified as true')
       const param = { UserPoolId: cognito.userPoolId, Username: name, UserAttributes: [{ Name: 'email_verified', Value: 'true' }] }
       await promisify(this.cognitoClient.adminUpdateUserAttributes.bind(this.cognitoClient, param))()
+
```

Comment:
move the `adminCreateUser`, `adminSetUserPassword`, `adminUpdateUserAttributes` to cognito service

Reply:
done